src/cloudant/design_document.py

```diff
@@ -75,6 +75,44 @@ def validate_doc_update(self):
         return self.get('validate_doc_update')
 
     @property
+    def filters(self):
+        """
+        Provides an accessor property to the filters dictionary in the locally cached
+        DesignDocument.  Filter functions enable you to add tests for filtering each
+        of the objects included in the changes feed.  If any of the function tests fail,
+        the object is either removed or filtered from the feed.  If the function
```

Comment:
This is a little confusing, if the function returns false how is it filtered or removed from the feed. Perhaps this is better to say `If any of the function tests fail, the object is filtered from the feed`.